pkg/csi/service/common/constants.go

```diff
@@ -400,4 +400,6 @@ const (
 	TopologyAwareFileVolume = "topology-aware-file-volume"
 	// PodVMOnStretchedSupervisor enables Pod Vm Support on stretched supervisor cluster
 	PodVMOnStretchedSupervisor = "podvm-on-stretched-supervisor"
+	// Storage-Quota-M2 enables support for snapshot quota feature
+	StorageQuotaM2 = "storage-quota-m2"
```

Comment:
Can we just call this snapshot-quota? We shouldn't have "m2" in the feature state name.

Reply:
ok will rename it to snapshot-quota